src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -19,11 +19,21 @@ import { DefaultTooltip } from 'cbioportal-frontend-commons';
 import GroupComparisonStore from '../GroupComparisonStore';
 import { GeneOptionLabel } from 'pages/resultsView/enrichments/EnrichmentsUtil';
 import { SingleGeneQuery } from 'shared/lib/oql/oql-parser';
+import { PathwayMapParams } from '../GroupComparisonPage';
 
 interface IGroupComparisonPathwayMapperProps {
     activeGroups: ComparisonGroup[] | undefined;
     genomicData: AlterationEnrichmentRow[];
     store: GroupComparisonStore;
```

Comment:
rename this to `groupComparisonStore` in order to distinguish from `userSelectionStore`